specs-go/config.go

```diff
@@ -40,13 +40,12 @@ type Process struct {
 	// Cwd is the current working directory for the process and must be
 	// relative to the container's root.
 	Cwd string `json:"cwd"`
-	// Capabilities are Linux capabilities that are kept for the container.
+	// Capabilities are Linux capabilities that are kept for the container. (this field is platform dependent)
 	Capabilities []string `json:"capabilities,omitempty" platform:"linux"`
-	// Rlimits specifies rlimit options to apply to the process.
-	Rlimits []Rlimit `json:"rlimits,omitempty"`
-	// NoNewPrivileges controls whether additional privileges could be gained by processes in the container.
-	NoNewPrivileges bool `json:"noNewPrivileges,omitempty"`
-
+	// Rlimits specifies rlimit options to apply to the process. (this field is platform dependent)
+	Rlimits []Rlimit `json:"rlimits,omitempty" platform:"linux,solaris"`
+	// NoNewPrivileges controls whether additional privileges could be gained by processes in the container. (this field is platform dependent)
+	NoNewPrivileges bool `json:"noNewPrivileges,omitempty" platform:"linux,solaris"`
```

Comment:
@anuthan - Could you comment on the use (or lack of) of `Rlimits` and `NoNewCapabilities` on Solaris? Thanks :smile: